README.md

```diff
@@ -30,3 +30,158 @@ If you have any questions or ideas feel free to join our <a href="https://discor
 # The license
 
 DXP Components is completely free and released under the Apache v2.0 License. Check <a href="https://github.com/hotwax/dxp-components/blob/main/LICENSE" target="_blank">LICENSE</a> for more details.
+
+# Components
+## DxpImage 
+### Introduction
+<li> dxpImage component is designed to handle the loading and display of images.</li>
+<li> It provides functionality to efficiently manage different types of image sources, including local assets, external web URLs, and images from a resource server.</li>
+
+### Basic Usage 
+The component dynamically loads and displays the specified image, considering different source scenarios, such as local assets, web URLs, or resource server images.
+
+To use the DxpImage component, integrate it into your Vue 3 application and provide the image source (src).
+
+      <template>
+        <img :src="imageUrl" />
+      </template>
+
+      <script lang="ts">
+      import { DxpImage } from "@hotwax/dxp-components";
+      </script>
+
+### Props
+#### 1. src
+<table>
+    <thead>
+      <tr>
+        <th>Prop Name</th>
+        <th>Description</th>
+        <th>Type</th>
+        <th>Required</th>
+        <th>Default Value</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><code>src</code></td>
+        <td>It's specifying that the component expects a prop named src.n this component props.src to access the value of the src.</td>
+        <td>string</td>
+        <td>False</td>
+        <td>‘context.defaultImgUrl’</td>
+      </tr>
+    </tbody>
+  </table>
+
+
+
+## Recommendation
+<li>Use the component to load and display images stored as local assets within the project directory.</li>
+<li>Employ the component to fetch and showcase images from external web URLs.</li>
+<li>Use the component to display images fetched from a resource server.</li>
+<li>Component's dynamic nature, allowing it to adapt different image sources based on the provided <b>props.src</b>. This is useful in scenarios where image sources can vary at runtime.</li>
+
+## DxpOmsInstanceNavigator 
+### Introduction
+<li> This component focuses on displaying information about the Order Management System (OMS) instance that the user is currently connected to.</li>
+
+### Basic Usage
+It is used to navigate to the OMS instance from the application. 
+
+To use the DxpOmsInstanceNavigator component, integrate it into your Vue 3 application.
+
+    <template>
+    <DxpOmsInstanceNavigator />
+    </template>
+
+    <script>
+    import { DxpOmsInstanceNavigator } from "@hotwax/dxp-components";
+    </script>
+
+###  Events 
+#### 1. goToOms()
+<li>The purpose of goToOms is to navigate the user to a different screen or page within the application that corresponds to a specific OMS instance.
+<li>It takes two parameters: <b>token</b> (an authentication token) and <b>omsName</b> (the name of the OMS instance).
```

Comment:
The example we have given above does not include these parameters. Are these given as props? are these values expected in a state somewhere?